src/unitTest/MultiInsertIntoDatabaseTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+include_once '../dataBase/XmlFeedModel.php';
+include_once '../dataBase/Database.php';
+
+use PHPUnit\Framework\TestCase;
+
+
+class MultiInsertIntoDatabaseTest extends TestCase
+{
+    protected $connectDB;
+    protected $insertData;
+    protected $xmlFeedObject;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->connectDB     = new Database();
+        $this->insertData    = new InsertData();
+        $this->xmlFeedObject = new XmlFeedModel();
+    }
+
+    public function testMultiInsertIntoDatabase(): void
```

Comment:
What happen if someone provide empty $collection? Or if it is array of numbers/string and other data types ?